cicd/gitops/viewing-argo-cd-logs-using-the-kibana-dashboard.adoc

```diff
@@ -0,0 +1,22 @@
+:_content-type: ASSEMBLY
+[id="viewing-argo-cd-logs-using-the-kibana-dashboard"]
+= Viewing Argo CD logs using the Kibana dashboard
+include::_attributes/common-attributes.adoc[]
+:context: viewing-argo-cd-logs-using-the-kibana-dashboard
+
+toc::[]
+
+You can view the Argo CD logs with {logging-title}. The {logging} provides a visualization of logs using Kibana dashboard. To integrate Argo CD with {logging}, the default configuration for the OpenShift Logging Operator enables logging with Argo CD.
```

Comment:
I'm not acquainted with this topic, so here is only a tentative suggestion for how to simplify this paragraph and add a few missing definitive articles:
```suggestion
You can view the Argo CD logs with the {logging-title}. The {logging} visualizes the logs on a Kibana dashboard. The OpenShift Logging Operator enables logging with Argo CD by default.
```

Reply:
This entire paragraph of intro content should be moved from here into the procedure module .adoc file. The assembly file should look like this:

```
:_content-type: ASSEMBLY
[id="viewing-argo-cd-logs-using-the-kibana-dashboard"]
= Viewing Argo CD logs using the Kibana dashboard
include::_attributes/common-attributes.adoc[]
:context: viewing-argo-cd-logs-using-the-kibana-dashboard

toc::[]

include::modules/gitops-storing-and-retrieving-argo-cd-logs.adoc[leveloffset=+1]

[role="_additional-resources"]
[id="additional-resources_viewing-argo-cd-logs-with-the-kibana-dashboard"]
== Additional resources
* xref:../../logging/cluster-logging-deploying.adoc#cluster-logging-deploy-console_cluster-logging-deploying[Installing the {logging-title} using the web console]
```